### DISCUSSION TITLE
I can't get past killing cros_camera_server on root escalation

### DISCUSSION THREAD

**Discussion by ppcheese56** [3kh0/ext-remover#433] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I just continue to get the error "/dev/fd/62: line 77: /proc/self/fd/4: cannot execute binary file: Exec format error"

**Comment by Aka-but-nice** [comment, marked as answer]
Try redoing it again, it may be because your usb, might be corrupted.

**Reply by ppcheese56** [reply]
i am already downgraded to v91 though, how would this have to do with my USB?
